.cci.jenkinsfile

```diff
@@ -1,6 +1,6 @@
-@Library('github.com/jlebon/coreos-ci-lib@master') _
+@Library('github.com/coreos/coreos-ci-lib@master') _
 
-coreos.pod([image: 'registry.fedoraproject.org/fedora:30', privileged: true, kvm: true]) {
+coreos.pod([image: 'registry.fedoraproject.org/fedora:30', runAsUser: 0, kvm: true, memory: "8Gi"]) {
```

Comment:
I think we need some headroom here.  How about 9Gi?

Reply:
Done. I was relying on the forgiveness of the scheduler, but yeah safer to just bump it up.